mqtt/src/main/scala/akka/stream/alpakka/mqtt/model.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.mqtt
+
+import scala.compat.java8.OptionConverters._
+
+final class MqttMessage private (val topic: String,
+                                 val payload: akka.util.ByteString,
+                                 val qos: Option[MqttQoS],
+                                 val retained: Boolean) {
+
+  /** Java API */
+  def getTopic: String = topic
```

Comment:
Is this for backwards compatibility? Usually we would not add getters for attributes that have java-friendly types.

Reply:
I was a bit indifferent there, I believe we don't need Java getters at all for messages filled by the user. But for messages created by the connector we might support Java getters for all fields, I guess.